terraform-example/README.md

```diff
@@ -0,0 +1,47 @@
+## This example and documentation are for the Beta v.1 of the Terraform Repositories, available exclusively to JFrog Beta customers.
+
+# Terraform Example
+
+## Overview
+This example demonstrates how to publish terraform modules to Artifactory.
+
+## Before Running the Example
+### Set Up the Environment
+1. Create local repository in Artifactory.
+2. Make sure version 2.12.0 or above of [JFrog CLI](https://jfrog.com/getcli/) is installed.
+
+### Validate the Setup
+In your terminal, validate that the following commands work.
+```console
+Output JFrog CLI version:
+> jfrog --version
+```
+
+### Scanning filesystem and packing modules
```

Comment:
1. I think that this section should be put last in the README. Let's first shiw the user how to run the example, only only then expalin how this all works behind the scenes.
2. Let's change the level of this section title from ### to ## - when you see the entire doc structure after the changes, you'll understand why.
3. Let's rename the section title to "How are the modules packed and published?"